base_comment_template/migrations/12.0.2.0.0/post-migrate.py

```diff
@@ -0,0 +1,62 @@
+import logging
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+
+    _logger.debug('Migrating res.partner comment_template_id')
+
+    cr.execute("""
```

Comment:
You can use `openupgrade.column_exists` for simplicity

Reply:
Yeah, just realized I could've used standard odoo tools for this 👍 there's already a column_exists there